homeassistant/components/light/flux_led.py

```diff
@@ -174,17 +180,23 @@ def name(self):
     @property
     def is_on(self):
         """Return true if device is on."""
-        return self._bulb.isOn()
+        if self.available:
```

Comment:
This is not necessary. When `available` returns False, we never fetch the state or any attributes.